modules/social_features/social_profile/modules/social_profile_organization_tag/social_profile_organization_tag.install

```diff
@@ -157,3 +159,44 @@ function social_profile_organization_tag_update_130001(): void {
   user_role_grant_permissions('contentmanager', ['create terms in profile_organization_tag']);
   user_role_grant_permissions('sitemanager', ['create terms in profile_organization_tag']);
 }
+
+/**
+ * Create entity-form-display or hide path field from Organization Tag taxonomy.
+ */
+function social_profile_organization_tag_update_130002(): void {
+  $storage = \Drupal::entityTypeManager()->getStorage('entity_form_display');
+  $form_display = $storage->load('taxonomy_term.profile_organization_tag.default');
+
+  // When entity-form-display isn't be created,
+  // it'll create from install folder.
+  if (empty($form_display)) {
+    // Get file from install folder and check if it exists.
+    $config_file = \Drupal::service('extension.list.module')->getPath('social_profile_organization_tag') . '/config/install/core.entity_form_display.taxonomy_term.profile_organization_tag.default.yml';
+    if (file_exists($config_file) === FALSE) {
+      \Drupal::logger('social_profile')->error('Configuration file for entity-form-display "profile_organization_tag" not found.');
+      return;
+    }
+
+    // Apply the configuration to create entity-form-display.
+    $view_config = Yaml::parseFile($config_file);
+    \Drupal::configFactory()->getEditable('core.entity_form_display.taxonomy_term.profile_organization_tag.default')
+      ->setData($view_config)
+      ->save();
+  }
+  else {
+    // Get fields from entity-form-display and check if path already be hidden.
```

Comment:
english update: `// Get fields from entity-form-display and check if path is already hidden.`